<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>522</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Documentation</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="cs8ProgramHeaderView" name="widget" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Parameters</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="cs8ProgramDataView" name="tableViewPars"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Variables</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="cs8ProgramDataView" name="tableViewVars"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="formVariableDetailEditor" name="detailEditor" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEditCode">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="menuRecent"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="listViewProgams"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cs8ProgramDataView</class>
   <extends>QTableView</extends>
   <header>cs8programdataview.h</header>
  </customwidget>
  <customwidget>
   <class>formVariableDetailEditor</class>
   <extends>QWidget</extends>
   <header>formvariabledetaileditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cs8ProgramHeaderView</class>
   <extends>QWidget</extends>
   <header>cs8programheaderview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableViewPars</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>editParameterDetail(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableViewVars</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>editParameterDetail(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editParameterDetail(QModelIndex)</slot>
 </slots>
</ui>
